packages/ui/src/components/Molecules/SearchForm.tsx

```diff
@@ -1,12 +1,13 @@
 import { useIntl } from '@amazeelabs/react-intl';
-import { useLocation } from '@custom/schema';
+import { TermContentHub, useLocation } from '@custom/schema';
 import { zodResolver } from '@hookform/resolvers/zod';
 import React from 'react';
 import { useForm } from 'react-hook-form';
 import { z } from 'zod';
 
 const formValueSchema = z.object({
   keyword: z.string().optional(),
+  terms: z.string().optional(),
```

Comment:
We could add a type constraint here to make sure the zod schema always matches `OperationVariables<typeof ContentHubQuery>`. I think it would look something like:
```typescript
const formValueSchema : z.ObjectType<OperationVariables<typeof ContentHubQuery>> = ...;
```

Reply:
I had to try it myself to remember how it works :) See my next PR.
If you would review that one, I think we are good to merge!